### PR TITLE
adding new caption

### PR DESCRIPTION
**Pull request by emrakblt**

### CODE REVIEW

File: _slides/06-emrakblt.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Champion Besiktas
```
Comment (egencer) [inline _slides/06-emrakblt.md:6]: That's kinda true! 👍

File: _slides/06-emrakblt.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (sennuur) [inline _slides/06-emrakblt.md:5]: looks okay

**Review by sennuur — APPROVED** [review]

File: _slides/06-emrakblt.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (egencer) [inline _slides/06-emrakblt.md:3]:
```suggestion
title: "Welcome to our slide deck!" Great 💯 
```

**Review by egencer — APPROVED** [review]
Ready to ship 🚢

**Pull request merged**